content/pt-br/docs/reference/glossary/statefulset.md

```diff
@@ -0,0 +1,22 @@
+---
+title: StatefulSet
+id: statefulset
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/statefulset/
+short_description: >
+  Gerencia deployment e escalonamento de um conjunto de Pods, com armazenamento duravel e indentificadores persistentes para cada Pod.
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+- storage
+---
+ Gerencia o deployment e escalonamento de um conjunto de {{< glossary_tooltip text="Pods" term_id="pod" >}}, *e fornece garantias sobre a ordem e unicidade* desses Pods.
+
+<!--more-->
+
+Como o {{< glossary_tooltip term_id="deployment" >}}, um *StatefulSet* gerencia Pods que são baseados em uma especificação de container idêntica. Diderente do Deployment, um *StatefulSet* mantém uma identitidade fixa para cada um de seus Pods. Esses pods são criados da mesma especificação, mas não são intercambiáveis: cada tem uma identificação persistente que se mantém em qualquer reagendamento.
```

Comment:
Typo in "diderente".

Reply:
In the last sentence ("cada tem uma identificação..."): perhaps use "cada um" instead of just "cada"?